### ISSUE TITLE
Training CIFAR-10 unusually fast?

### ISSUE THREAD

**Issue by tarunn2799** [ariG23498/G-SimCLR#12]
Hey so I tried using your notebook where you use a vanilla simclr model to train on the CIFAR10 dataset to train a model, but I noticed that the model converges unusually fast. CIFAR10 training took <5-10 mins to train running the same code here on colab. 

Using a resnet to train CIFAR10 usually takes at least 30-40 mins on colab, is there a reason how this is converging so quickly? Training CIFAR-10 on the official simclr code takes much longer as well.

**Comment by ariG23498** [comment]
Hey @tarunn2799 
I would like to bring into your notice that we did not particularly use the exact same notion of SimCLR. If you look into our research paper you will see that we have actually built a **minimalistic SimCLR model**. The minimalistic approach is the reason why you see the training converge quick.

**Comment by jmarrietar** [comment]
Hi @ariG23498 . Reading the paper it seems that the modifications are: 

- We use a subset of augmentation operations used in SimCLR.
- the optimizer used is stochastic gradient descent with a cosine learning rate decay instead of LARS

Is there other notable modification to take into account?. 

Thanks!

**Comment by sayakpaul** [comment]
The modifications are in how we formulate the batches for SimCLR training to impose pseudo-label knowledge inside of them. I think the abstract makes that clear.
